### PR TITLE
Project 5 - Design Teardown

### PR DESCRIPTION
**Pull request by expjazz**
The project involves putting elements into the right position and focusing on mapping the UX and UI of the original Smashing Magazine Website.

We used the grid system, also flexbox for positioning. For the background, we used scales of grey, with as darker the color more important is the item. 

We also imported a font from the Google Fonts in which we think that it suited the project better.

### CODE REVIEW

**Comment by thsvr** [comment]
### **STATUS: CHANGES REQUIRED** :+1: :+1: 

Hi @expjazz and @praz99 :smile_cat: :smile_cat: 


Your page looks beautiful and you guys are doing a great job :fireworks:  So you guys are almost there! **Can you please read carefully the required changes below? They are necessary to be implemented on the project** :+1: 

:v: In this project, article tag has to be used for each article. [Here](https://www.w3schools.com/tags/tag_article.asp) you guys can read and understand more about it. Can you please fix it?

:v: [Here](https://www.w3schools.com/tags/tag_footer.asp) you guys can read about footer tag and where it is important to be implemented. Can you please consider it?

:v: Can you please update the live version on readme? 

**SUGGESTION FOR THE NEXT PROJECTS:**

:v: You can create a feature branch with a more descriptive name on the next projects, like ''main-page'' or ''greyscale-page'', for example. It will be more descriptive. :+1: :+1: 

If you have any questions, don't hesitate to contact me through slack[ @thaís Vieira Azevedo](url)


### **Happy coding and good luck on the next review** :computer: :computer: 

_Project reviewed by Thaís Vieira (TSE)_

**Comment by mikenath223** [comment]
**Pull request currently under review by Michgolden Ukeje (TSE).**
Kindly exercise patience 🐱

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team👋

You've worked quite hard on this @expjazz and @praz99 👨‍💻👏

However, there are a few changes that need to be made before proceeding.
**Hang in there, we've got you.💪**



- [ ] Required - There are differences between the original webpage and yours, kindly ensure you clone the visual hierarchy of the page, with the way the templates look. Consider this:

**THIS IS THE WAY THE ORIGINAL LOOKS ON THE SECTION ALMOST TO THE MIDDLE OF THE PAGE**👇

ice_screenshot_20200402-101239

**THIS IS THE WAY THE NEXT SECTION THAT FOLLOWS IT LOOKS LIKE**👇
ice_screenshot_20200402-101250

**FIX THIS ISSUES IN YOURS, CLONING THE VISUAL LOOK OF IT THE TEXT CONTENT DOES'T MATTER.**

- [ ] Required - The newsletter section just before the footer completely fills the full-width of the page.and also the section directly above it contains four(4) templates positioned side-by-side, ensure you have the same on yours too.

**THIS IS THE NEWSLETTER SECTION JUST BEFORE THE FOOTER👇**
ice_screenshot_20200402-094526

**THIS IS THE FOUR(4) TEMPLATES SECTIONS THAT COME BEFORE IT**👇
ice_screenshot_20200402-103904

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAVE A SMASHING CODING DAY! 👨‍💻👋**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+  align-self: center;
+  font-size: 14px;
+}
+
+.footer-list {
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-rows: repeat(3, 1fr);
+  padding: 0 150px;
+}
```
Comment (sumancrest0001) [inline style.css]: We do not have to use a grid. Just make keep it simple and align all the items at the center of the footer box.

**Comment by sumancrest0001** [comment]
### Status: Required a change

Great job TEAM :clap: :clap: 
Everything looks great but one small change is required to get :+1: :+1:  for this project.

It is quite simple but we have to tick all the requirements. Please make the change and submit for another review.
Screenshot from 2020-04-02 19-27-33

**It should be like this :point_down:** 
Screenshot from 2020-04-02 19-27-17

**Comment by expjazz** [comment]
As required by the last review, we made the necessary adjustments to the footer.

**Review by men32z — APPROVED** [review]
## Good job making the changes, you can merge.

BlgS1sdIUAEybkW

### Status Approved

**Pull request merged**